src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class FileWriterImplTest {
+    private FileWriter fileWriter;
+

```

Comment:
The FileWriter object should be instantiated before each test. Consider using @BeforeEach annotation to initialize it.

Reply:
ok, done